lib/cucapp/Resources/features/support/Cucumber+Extensions.j

```diff
@@ -16,4 +16,20 @@
     return '{"result" : "__CUKE_ERROR__"}';
 }
 
-@end
\ No newline at end of file
+- (CPString)delegatePropertyIsEqual:(CPArray)params
+{
+    var property = params[0];
+
+    if (property == nil)
+        return '{"error" : "The property ' + property +' was not found"}';
+
+    var expected_value = params[1],
+        current_value = [[[CPApplication sharedApplication] delegate] valueForKey:property];
+
+    if (current_value != expected_value)
+        return '{"error" : "The value should be ' + expected_value +' but was '+ [current_value description] + '"}';
+
+    return '{"error" : "__NO_ERROR__"}';
+}
+
+@end
```

Comment:
There isn't any convention or something, we are used to use `'{"result" : "OK"}';` or `'{"result" : "__CUKE_ERROR__"}';`
I think it would be better to continue like this for consistency